docs/reference/contributing/guidelines/workflow.md

```diff
@@ -107,11 +107,13 @@ Every pull request changing or adding functionality must contain a release notes
 It must contain:
 
 - A brief description of changes introduced.
```

Comment:
Is i intentional that this section (for functional changes) differs from the section for breaking changes?

We have:

- "- A brief description of changes introduced, including justification description." for breaking changes
- "- A brief description of changes introduced." for functional changes

Should this be:

- "- A brief description of changes introduced, including justification description." for breaking changes
- "- A brief description of changes introduced, including justification description." for functional changes

Reply:
it was, but reading the proposal. I fixed it in b686636